modules/engine/package.json

```diff
@@ -35,14 +35,15 @@
   ],
   "sideEffects": false,
   "scripts": {
-    "build-bundle": "ocular-bundle ./src/index.ts",
-    "pre-build": "npm run build-bundle && npm run build-bundle -- --env=dev"
+    "build-minified-bundle": "ocular-bundle ./src/index.ts -output=dist/dist.min.js",
+    "build-dev-bundle": "ocular-bundle ./src/index.ts -output=dist/dist.dev.js -- --env=dev",
+    "prepublishOnly": "npm run build-minified-bundle && npm run build-dev-bundle"
+  },
+  "peerDependencies": {
+    "@luma.gl/core": "^9.0.0-beta.4",
+    "@luma.gl/shadertools": "^9.0.0-beta.4"
   },
   "dependencies": {
-    "@babel/runtime": "^7.0.0",
-    "@luma.gl/constants": "9.0.0-beta.4",
-    "@luma.gl/core": "9.0.0-beta.4",
-    "@luma.gl/shadertools": "9.0.0-beta.4",
```

Comment:
Question – When something is a `peerDependency` of a package, I'd generally expect to see it as a `devDependency` of either the package itself, or the workspace root if workspaces are involved. Is it OK and intended that `@luma.gl/*` peer dependencies appear in neither of those places?

Reply:
You may be right. Perhaps it is the fact that the examples are part of the workspace that gets these installed right now.

That should probably be addressed for the long term. I can tinker with that after this lands, would need to remove examples make sure an install failure is triggered and then add these in appropriate places.

Reply:
I'm fine with whatever you decide on this! Just wasn't sure whether some build tools might expect to see dependencies in one of those places. Maybe something to leave as-is for now and revisit if we run into build issues.